README.md

```diff
@@ -61,33 +61,23 @@ gitlab.subdomain.example.com
 ...
 ```
 
-## Clone the repository
-
-Clone the repository to have the latest `main` branch content
+## Getting the binary
 
 ```bash
-# via HTTPS
-git clone https://github.com/kubefirst/kubefirst.git
-
-# via SSH
-git clone git@github.com:kubefirst/kubefirst.git
-```
+#Check the release page:
+#https://github.com/kubefirst/kubefirst/releases
 
-## Start the Container
+export KUBEFIRST_VERSION=1.8.5
```

Comment:
i think we may want this to read 1.8.6 and update the config.Version to match for the 1.8.6 publish operation to align after merge